sparse/impl/KokkosSparse_sptrsv_solve_impl.hpp

```diff
@@ -220,13 +220,13 @@ struct SptrsvWrap {
 
       // copy-assignment
       KOKKOS_INLINE_FUNCTION
-      ArrayType& operator=(const ArrayType& rhs_) {
-          if (this != &rhs_) {
-              for (size_type i = 0; i < MAX_VEC_SIZE; ++i) {
-                  m_data[i] = rhs_.m_data[i];
-              }
+      ArrayType &operator=(const ArrayType &rhs_) {
+        if (this != &rhs_) {
+          for (size_type i = 0; i < MAX_VEC_SIZE; ++i) {
+            m_data[i] = rhs_.m_data[i];
           }
-          return *this;
+        }
+        return *this;
```

Comment:
what happened here?

Reply:
yes this is formatting related, I’m not sure why it wasn’t formatted before.